testdata/chain/test_migrate.py

```diff
@@ -19,20 +20,24 @@ def Main(operation, args):
     if operation == "name":
         return NAME
 
+
 def MigrateContract(code):
     """
     Note that the existing contract will be replaced by the newly migrated contract
     :param code: your avm code
     :return:
     """
-    sucess = Migrate(code, True, "name", "version", "author", "email", "description")
-    print("Migrate successfully")
-    Notify(["Migrate successfully"])
-    return True
+    success = Migrate(code, True, "name", "version", "author", "email", "description")
```

Comment:
here Migrate return is interface can not convert to bool. reverted is ok.

Reply:
Fixed this